src/controllers/arraysControllers.js

```diff
@@ -0,0 +1,49 @@
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./../lib/arrays');
+
+exports.elementAtID = (req, res) => {
+  const { array } = req.body;
+  const index = req.params.index;
```

Comment:
any reason why you didn't deconstruct for index too? Good to do both the same for consistency.